pkg/tfgen/docs.go

```diff
@@ -1767,6 +1764,34 @@ func (g *Generator) convertHCL(e *Example, hcl, path string, languages []string)
 	isCompleteFailure := len(failedLangs) == len(languages)
 
 	if isCompleteFailure {
+		languagesByErrMsg := map[string]map[string]struct{}{}
+		for lang, convertErr := range failedLangs {
+			m := convertErr.Error()
+			if _, ok := languagesByErrMsg[m]; !ok {
+				languagesByErrMsg[m] = map[string]struct{}{}
+			}
+			languagesByErrMsg[m][lang] = struct{}{}
+		}
+
+		var err error
+
+		seen := map[string]struct{}{}
+		for _, convertErr := range failedLangs {
+			if _, dup := seen[convertErr.Error()]; dup {
+				continue
+			}
+			m := convertErr.Error()
+			seen[m] = struct{}{}
+
+			langs := []string{}
+			for l := range languagesByErrMsg[m] {
+				langs = append(langs, l)
+			}
+			sort.Strings(langs)
+			ls := strings.Join(langs, ", ")
+			err = multierror.Append(err, fmt.Errorf("[%s] %w", ls, convertErr))
```

Comment:
at first glance, this looks like we're not just skipping duplicate errors but also consolidating different errors?

Reply:
Right.